ignition-math/PKGBUILD

```diff
@@ -12,10 +12,10 @@ depends=('gcc-libs')
 makedepends=('ignition-cmake>=2')
 optdepends=('eigen')
 conflicts=()
-source=("https://bitbucket.org/ignitionrobotics/ign-math/get/${pkgname}6_${pkgver}.tar.bz2")
-sha256sums=('5463d2ee5202ad60965a613361c1b12e34d3a8c49ad9037b16120f89b0250745')
+source=("https://github.com/ignitionrobotics/ign-math/archive/master.tar.gz")
```

Comment:
Instead of using master we package the latest release https://github.com/ignitionrobotics/ign-math/releases/tag/ignition-math6_6.4.0

Also based on the warning here: https://wiki.archlinux.org/index.php/PKGBUILD#source
Do you mind adding `${pkgname}-${pkgver}::` to the beginning of the source url?